src/sausage-rolls.md

```diff
@@ -0,0 +1,25 @@
+# Sausage Rolls
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 30-60 min
+
+## Ingredients
+
+- Eggs
+- Puff pastry
+- Pork meat / sausage meat
+
+## Directions
+
+1. Place baking paper onto a tray
+2. Whisk eggs
+3. Lay out pastry, brush or pour egg whisk so it lightly covers the pastry
+4. Prepare sausage meat onto pastry
+5. Fold pastry around the meat, apply extra egg whisk to the 'seal' where the pastry is folded together
+6. Bake until golden brown.
```

Comment:
Think you should elaborate on the directions here. How long should I whisk eggs for? What should the result of whisking the eggs look like?